### ISSUE TITLE
Custom error message for filename-naming-convention rule

### ISSUE THREAD

**Issue by kopach** [dukeluo/eslint-plugin-check-file#35]
Hi, is there any chance to provide custom error message rule for `filename-naming-convention`?
Currently, especially when used with `prefined match` syntax - error message is very misleading/unclear
We would get error like

```sh
 error  The filename "MyComponent.tsx" does not match the "<1>" pattern  check-file/filename-naming-convention
```
What I would like to have instead is the ability to define custom error message (even hard coded, without dynamic parts like exact file name or so)

Error like this would be much more helpful

```sh
 error  The filename doesn't match file naming convention defined for this project. See contribute.md for details
```

**Comment by dukeluo** [comment]
The rule `filename-naming-convention` is a little different with rule `filename-blocklist`. If we want to support custom error message for rule `filename-naming-convention`, maybe we can do like this https://github.com/lo1tuma/eslint-plugin-mocha/pull/206/files

**Comment by dukeluo** [comment]
@kopach https://github.com/dukeluo/eslint-plugin-check-file/blob/main/docs/rules/filename-naming-convention.md#errormessage It has been released in v2.7.0.
